vtex-sites/gatsby.store
# chore: Uses CSS Modules on product components at PLP

## What's the purpose of this pull request?

This is the first PR to accomplish the use of CSS Modules on all components at PLP. [FSSS-355](https://vtex-dev.atlassian.net/browse/FSSS-355).

This PR does this for the `Product` and `ProductGrid` components.

## How does it work?

These steps were followed to accomplish it for both components:

- Change CSS file name to `.module.scss`
- Wrap css content into a single class
- Import css at the component
- Removes import in all other places

## How to test it?

Enter the preview and goes to the PLP, Search, and homepage. Everything should be working fine.

## Checklist

<em>You may erase this after checking them all ;)</em>



- PR description
